dcrpool.go

```diff
@@ -139,8 +138,12 @@ func realMain() error {
 		}
 	}()
 
+	// Get a context whose done channel will be closed when a shutdown signal
```

Comment:
Theres a couple of lines at the start of `realMain`  which can be removed, as well as a global declaration of `signals` just under the package imports.